### DISCUSSION TITLE
[Request] Adding a maxdmg command (like maxhp)

### DISCUSSION THREAD

**Discussion by Flexxxxx** [Zrips/CMI#1603] [Feature Requests]
**Description of issue or feature request:**
There are a few players on our server who have heightened base damage stats which aren't being given by items (we believe it could be a rare occurrence from /cmi invload). Since /cmi maxhp is already a command, I was wondering if it was possible to create a similar command, except for the damage stat which would be /cmi maxdmg or something.

Hope this is possible! Thanks for the great plugin :) 
---

**Comment by zykrah** [comment]
upvote

**Comment by Zrips** [comment]
@Flexxxxx This atleast needs more clarification. What you mean by max damage? Amount of damage they could do to some one else? And CMI doesn't have a code to change players base damage values. You could find player with strange damage, find his player data file, open with something like NBTExplorer and check Atributtes where it says attackDamage. Default is 1.
For feature itself. It could simply be the case of cheat usage which would allow to send bunch of attack packets in short time, which would look like player has huge attack damage while he has normal one.

**Comment by Flexxxxx** [comment]
Yeah, sorry I did mean attackDamage. Just a way to edit a users attackDamage through a command in-game would be incredible. The feature could also have other uses too, such as a rank being given increased base damage as a perk, punishments, etc, which would all be achieved with such a command.
